mep/books/views.py

```diff
@@ -64,7 +64,7 @@ def get_form(self, *args, **kwargs):
 
     def get_queryset(self):
         # NOTE faceting on format as placeholder, no UI yet
-        sqs = WorkSolrQuerySet().facet_field('format')
+        sqs = WorkSolrQuerySet().facet_field('format', exclude='format')
```

Comment:
Do we need this?

Reply:
probably not since we're not even using the facet data, but for some reason I thought we wanted to facet on at least one thing even when not using it? I can take it out though

Reply:
The only MVP filter is circulation dates, and we don't facet on that. I think it's possible we'll get to format, but I'm inclined to remove it until we do.